frontend/src/app/components/admin/modals/admin-absence-edit-modal/admin-absence-edit-modal.component.ts

```diff
@@ -0,0 +1,195 @@
+import { Component, OnInit, Inject, OnDestroy } from "@angular/core";
+import { FormGroup, FormControl, Validators } from "@angular/forms";
+import { Absence, AbsenceType } from "src/app/models/Absence.model";
+import { Subject } from "rxjs";
+import { ApiCommunicationService } from "src/app/services/api-communication.service";
+import { MatDialogRef, MAT_DIALOG_DATA } from "@angular/material";
+import { takeUntil } from "rxjs/operators";
+import { User } from "src/app/models/User.model";
+import * as moment from "moment";
+
+@Component({
+  selector: "app-admin-absence-edit-modal",
+  templateUrl: "./admin-absence-edit-modal.component.html",
+  styleUrls: ["./admin-absence-edit-modal.component.css"]
+})
+export class AdminAbsenceEditModalComponent implements OnInit, OnDestroy {
+  private createAbsenceForm: FormGroup;
+  private types;
+  private error: string;
+  private absence: Absence;
+  private message = "Edit";
+  private _unsubscribe$ = new Subject<void>();
+  private leaders: User[];
+  private users: User[];
+  private isTypeEdit = { value: false };
```

Comment:
ittis ugyanaz a kérdés majd mint a másik PR-nél
```suggestion
  private isTypeEdit = false;
```